lib/puppetserver/ca/action/list.rb

```diff
@@ -57,6 +60,12 @@ def run(input)
           config = input['config']
           certnames = input['certname'] || []
           all = input['all']
+          output_format = input['format']
```

Comment:
So, I'm a little torn here, where right now we have an either in formats with a default vs how I feel like formats are often treated (where you may have many). I would probably go with making `output_format` have a default of 'text' like
```rb
output_format = input['format'] || 'text'
```
Then you don't have to check for `nil`, and by doing `case` or `if` / `elsif` blocks it makes it easy to add new formats.

If we didn't go that route and leaned into the either or nature of this, I would probably change the variable `output_format` to be something more like a json flag, eg
```rb
output_as_json = input['format'] == 'json'
```

Reply:
I like the first suggestion.  Eliminate the need to have check for `nil`.

Reply:
I think if we're going to have a `format` flag that takes an arg, we should set ourselves up so it'd be easy to add other args later (e.g. yaml).